app/src/main/res/values/strings.xml

```diff
@@ -154,6 +154,8 @@
     <string name="settings_advanced_reset_confirmation">Do you REALLY want to reset your safe? All data will be lost!</string>
     <string name="settings_advanced_backup_title">Backup</string>
     <string name="settings_advanced_backup_summary">Create a backup of your safe</string>
+    <string name="settings_advanced_delete_imported_title">Delete imported files</string>
```

Comment:
Please add those keys to all language files aswell (in english). and add ``<<!-- TODO -->` in the same line.
You might see some in those files. This is needed for the badges in the README to work

Reply:
Added the translations and added `<!-- TODO -->` to the end of the line.